src/material/list/list.scss

```diff
@@ -300,6 +300,10 @@ mat-action-list {
   outline: none;
 }
 
+.mat-list-item-disabled {
+  pointer-events: none;
```

Comment:
This would also disable tooltips, I think we'd want to avoid

Reply:
How would we prevent clicking on it though? We've done something similar to support disabled tabs in the tab nav bar.

Reply:
Yeah, the tabs thing is why I thought about this (someone mentioned it in a bug). I don't know of another way to do this other than `preventDefault` / `stopPropagation`

Reply:
We can't rely on `preventDefault`/`stopPropagation`, because we don't have a way of ensuring that our event listeners are invoked before the user's.